gradle_test.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/jfrog/jfrog-cli-go/artifactory/commands/gradle"
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-go/inttestutils"
+	"github.com/jfrog/jfrog-cli-go/utils/tests"
+)
+
+func TestGradleBuildWithServerID(t *testing.T) {
+	initBuildToolsTest(t)
```

Comment:
The *initBuildToolsTest* function should probably not exist, because the *buildtools_test.go* file will be removed, unless you think that we need one init function for all build tools. In that case, it this function should probably in the tests utils file.